### PR TITLE
i18n: update zh_cn.js

### PR DESCRIPTION
**Pull request by andelf**
- Fix wrong translations. esp. for emotions.
- use Chinese characters instead of unicode escape sequence.

### CODE REVIEW

**Comment by andelf** [comment]
This conflicts with #2791 . Project owner should choose one and close the other.

File: js/languages/zh_cn.js [hunk 1]
```diff
@@ -304,13 +304,13 @@ $.FE.LANGUAGE['zh_cn'] = {
...
     "Embed URL": "嵌入网址",
-    "Paste in a URL to embed": "粘贴在一个网址中嵌入",
+    "Paste in a URL to embed": "粘贴要嵌入的网址",
 
     // Word Paste.
-    "The pasted content is coming from a Microsoft Word document. Do you want to keep the format or clean it up?": "粘贴的内容来自微软Word文档。你想保留格式还是清理它？",
-    "Keep": "保持",
-    "Clean": "清洁",
-    "Word Paste Detected": "检测到字贴"
+    "The pasted content is coming from a Microsoft Word document. Do you want to keep the format or clean it up?": "粘贴的内容来自微软 Word 文档。你想保留格式还是清清除格式？",
```
Comment (PeterPanZH) [inline js/languages/zh_cn.js]: > 粘贴的内容来自微软 Word 文档。你想保留格式还是**清**清除格式？

多了个清

File: js/languages/zh_cn.js [hunk 2]
```diff
@@ -37,230 +37,230 @@
...
     // Place holder
-    "Type something": "\u8f93\u5165\u4e00\u4e9b\u5185\u5bb9",
+    "Type something": "输入内容",
 
     // Basic formatting
-    "Bold": "\u7c97\u4f53",
-    "Italic": "\u659c\u4f53",
-    "Underline": "\u4e0b\u5212\u7ebf",
-    "Strikethrough": "\u5220\u9664\u7ebf",
+    "Bold": "黑体",
```
Comment (PeterPanZH) [inline js/languages/zh_cn.js]: `加粗` 比较准确

File: js/languages/zh_cn.js [hunk 3]
```diff
@@ -37,230 +37,230 @@
...
+    "Drop image": "拖入图片",
+    "or click": "或点击",
+    "Manage Images": "管理图片",
+    "Loading": "加载中",
+    "Deleting": "删除中",
+    "Tags": "标签",
+    "Are you sure? Image will be deleted.": "图片将会被删除，是否确认？",
+    "Replace": "替换",
+    "Uploading": "上传中",
+    "Loading image": "加载图片",
```
Comment (Dafrok) [inline js/languages/zh_cn.js]: 图片加载中

File: js/languages/zh_cn.js [hunk 4]
```diff
@@ -37,230 +37,230 @@
...
-    "Align Left": "\u5de6\u5bf9\u9f50",
-    "Align Center": "\u5c45\u4e2d",
-    "Align Right": "\u53f3\u5bf9\u9f50",
-    "Align Justify": "\u4e24\u7aef\u5bf9\u9f50",
-    "None": "\u65e0",
+    "Align": "对齐",
+    "Align Left": "左对齐",
+    "Align Center": "居中",
+    "Align Right": "右对齐",
+    "Align Justify": "均齐",
```
Comment (Dafrok) [inline js/languages/zh_cn.js]: 两端对齐

**Comment by andelf** [comment]
thanks for your comments. 😆

**Comment by PeterPanZH** [comment]
Since there is already an open PR(#2791 ), I think you should merge your commits to that PR. Open another one is not a good choice. : )

**Comment by yenshih** [comment]
@stefanneculai 
This PR should definitely not be merged, since there's a number of ridiculous mistakes.
For example, this PR translates `Decrease Indent` and `Increase Indent` to the completely opposite meaning.
IMO, there's no reason to choose this instead of https://github.com/froala/wysiwyg-editor/pull/2791.

**Pull request closed without merging**